documents/coredocument.go

```diff
@@ -197,6 +202,47 @@ func (cd *CoreDocument) PrepareNewVersion(collaborators []string, documentPrefix
 
 }
 
+// PrepareNewVersion prepares the next version of the CoreDocument
```

Comment:
Let's keep the linting happy though:
```suggestion
// PrepareNewVersion1 prepares the next version of the CoreDocument
```